.github/workflows/test.yml

```diff
@@ -2,12 +2,15 @@ name: Test
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   test:
+    strategy:
+      matrix:
```

Comment:
I didn't know you could add the test matrix to github actions. This is cool.

Reply:
:)